tools/osbuilder/rootfs-builder/ubuntu/Dockerfile.in

```diff
@@ -27,10 +43,45 @@ RUN apt-get update && \
     make \
     makedev \
     multistrap \
+    musl \
+    musl-dev \
     musl-tools \
     protobuf-compiler \
-    xz-utils
+    xz-utils \
+    pip \
+    python3-dev \
+    xz-utils \
+    zstd && \
+    apt-get clean && rm -rf /var/lib/apt/lists/&& \
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain ${RUST_TOOLCHAIN}
+
+RUN ARCH=$(uname -m); \
+    goarch=""; \
+    kernelname=$(uname -s | tr '[:upper:]' '[:lower:]'); \
+    case "${ARCH}" in \
+        "aarch64") goarch="arm64" ;; \
+        "ppc64le") goarch=${ARCH} ;; \
+        "x86_64") goarch="amd64" ;; \
+        "s390x") goarch=${ARCH} ;; \
+        *) echo "Unsupported architecture: ${ARCH}" && exit 1 ;; \
+    esac; \
+    curl -OL "https://storage.googleapis.com/golang/go${GO_VERSION}.${kernelname}-${goarch}.tar.gz" && \
+    tar -C "${GO_HOME}" -xzf "go${GO_VERSION}.${kernelname}-${goarch}.tar.gz" && \
+    rm "go${GO_VERSION}.${kernelname}-${goarch}.tar.gz"
+
 # aarch64 requires this name -- link for all
 RUN if [ ! -f "/usr/bin/$(uname -m)-linux-musl-gcc" ]; then ln -s /usr/bin/musl-gcc "/usr/bin/$(uname -m)-linux-musl-gcc"; fi
 
-@INSTALL_RUST@
```

Comment:
If you are removing this, then do we need to clean up https://github.com/kata-containers/kata-containers/blob/c774cd6bb0c1a631df9892f425d3deb21a2a9ac2/tools/osbuilder/scripts/lib.sh#L250 which injects the rust install here?

The injected version also has proxies configured, so I'm not sure if we need to keep them?

Reply:
The proxy is set in rootfs.sh  we can remove this "old" code. 

```
generate_dockerfile "${distro_config_dir}"
		"$container_engine" build  \
			${engine_build_args} \
			--build-arg http_proxy="${http_proxy}" \
			--build-arg https_proxy="${https_proxy}" \
			--build-arg RUST_TOOLCHAIN="$(get_package_version_from_kata_yaml  "languages.rust.meta.newest-version")" \
			-t "${image_name}" "${distro_config_dir}"
```

Reply:
And the same is true for the `docker run` command